src/Illuminate/Database/Query/Builder.php

```diff
@@ -9,6 +9,7 @@
 use Illuminate\Support\Str;
 use InvalidArgumentException;
 use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
```

Comment:
Thanks for the PR, however, we cannot call out facades from the framework core like this I'm afraid.

Reply:
I just looked at the alias for `DB`. Feel free to change this.